### PR TITLE
Fix page layout of installing kubectl on linux

### PR DESCRIPTION
**Pull request by dipesh-rawat**
- Changes to fix the current broken page layout at https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/

- Possibly, resolves https://github.com/kubernetes/website/issues/42651

- [Original Page](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/) | [Deploy Preview](https://deploy-preview-42655--kubernetes-io-main-staging.netlify.app/docs/tasks/tools/install-kubectl-linux/)

### CODE REVIEW

File: content/en/docs/tasks/tools/install-kubectl-linux.md [hunk 1]
```diff
@@ -147,9 +147,9 @@ The following methods exist for installing kubectl on Linux:
    echo 'deb [signed-by=/etc/apt/keyrings/kubernetes-apt-keyring.gpg] https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/deb/ /' | sudo tee /etc/apt/sources.list.d/kubernetes.list
    ```
 
-   {{< note >}}
-   To upgrade kubectl to another minor release, you'll need to bump the version in `/etc/apt/sources.list.d/kubernetes.list` before running `apt-get update` and `apt-get upgrade`. This procedure is described in more detail in [Changing The Kubernetes Package Repository](/docs/tasks/administer-cluster/kubeadm/change-package-repository/).
-   {{< /note >}}
+{{< note >}}
+To upgrade kubectl to another minor release, you'll need to bump the version in `/etc/apt/sources.list.d/kubernetes.list` before running `apt-get update` and `apt-get upgrade`. This procedure is described in more detail in [Changing The Kubernetes Package Repository](/docs/tasks/administer-cluster/kubeadm/change-package-repository/).
+{{< /note >}}
```
Comment (tengqm) [inline content/en/docs/tasks/tools/install-kubectl-linux.md:152]: This note block should be indented by 3 spaces.

File: content/en/docs/tasks/tools/install-kubectl-linux.md [hunk 2]
```diff
@@ -170,27 +170,27 @@ In releases older than Debian 12 and Ubuntu 22.04, `/etc/apt/keyrings` does not
...
 {{< /note >}}
 
-1. Install kubectl using `yum`:
+2. Install kubectl using `yum`:
 
-```bash
-sudo yum install -y kubectl
-```
+   ```bash
+   sudo yum install -y kubectl
```
Comment (ehmad11) [inline content/en/docs/tasks/tools/install-kubectl-linux.md:192]: `yum` should be replaced by `dnf` for newer versions

**Review by ehmad11 — CHANGES_REQUESTED** [review]

File: content/en/docs/tasks/tools/install-kubectl-linux.md (hunk 2 above)
Comment (sftim) [reply content/en/docs/tasks/tools/install-kubectl-linux.md:192]: This PR's aim (“fix page layout”) doesn't cover a switch from `yum` to `dnf`. Let's not make that change in _this_ PR.

File: content/en/docs/tasks/tools/install-kubectl-linux.md [hunk 3]
```diff
@@ -170,27 +170,27 @@ In releases older than Debian 12 and Ubuntu 22.04, `/etc/apt/keyrings` does not
...
+   baseurl=https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/rpm/
+   enabled=1
+   gpgcheck=1
+   gpgkey=https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/rpm/repodata/repomd.xml.key
+   EOF
+   ```
 
 {{< note >}}
 To upgrade kubectl to another minor release, you'll need to bump the version in `/etc/yum.repos.d/kubernetes.repo` before running `yum update`. This procedure is described in more detail in [Changing The Kubernetes Package Repository](/docs/tasks/administer-cluster/kubeadm/change-package-repository/).
 {{< /note >}}
```
Comment (sftim) [inline content/en/docs/tasks/tools/install-kubectl-linux.md:187]: How should this be indented? (is it part of a list?)

Comment (dipesh-rawat) [reply content/en/docs/tasks/tools/install-kubectl-linux.md:187]: On indenting the note block, the tab contents are not generated properly.

indentation-issue

**Pull request merged**